.github/workflows/build-new-example-web.yml

```
@@ -0,0 +1,34 @@
+name: build-deploy-new-sdk-example
+
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      TEST_ENDPOINT: example.onekey-test.com
+      PROJECT_PATH: packages/connect-examples/new-example
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)